.github/workflows/verify_e2e-techdocs.yml

```diff
@@ -44,3 +44,5 @@ jobs:
       - name: techdocs-cli e2e test
         working-directory: packages/techdocs-cli
         run: yarn test:e2e:ci
+        env:
```

Comment:
honestly this one may be pretty redundant, but i thought i'd throw it in there anyway just to cover our bases